src/downloadstab.cpp

```diff
@@ -17,6 +17,9 @@ DownloadsTab::DownloadsTab(OrganizerCore& core, Ui::MainWindow* mwui)
   ui.list->setItemDelegate(new DownloadProgressDelegate(
     m_core.downloadManager(), ui.list));
 
+  ui.list->setDragEnabled(true);
```

Comment:
maybe it's better to put these in the ui file? since I don't think they are changed later, so we have the correct stuff in qtcreator

Reply:
actually it's already in the ui file so I just removed those